src/main/kotlin/uk/gov/justice/digital/hmpps/authorizationserver/data/model/Authorization.kt

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.digital.hmpps.authorizationserver.data.model
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.Id
+import jakarta.persistence.Table
+import java.time.Instant
+
+@Entity
+@Table(name = "oauth2_authorization")
+data class Authorization(
+  @Id private val registeredClientId: String,
```

Comment:
The database column called `id` is the primary key field
You should also provide mappings to the following columns:
`registered_client_id`
`principal_name`
`authorization_grant_type`

Reply:
Updated